pkg/client/results/processing_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright the Sonobuoy contributors 2019
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package results
+
+import (
+	"fmt"
+	"github.com/heptio/sonobuoy/pkg/plugin"
+	"github.com/heptio/sonobuoy/pkg/plugin/driver"
+	"github.com/heptio/sonobuoy/pkg/plugin/driver/job"
+	"github.com/heptio/sonobuoy/pkg/plugin/manifest"
+	"testing"
+)
+
+func TestProcess(t *testing.T) {
+	xmlFile := "xml1.xml"
+	getPlugin := func(pluginDriver, format, outputFile string) plugin.Interface {
+		return &job.Plugin{Base: driver.Base{
+			Definition: manifest.Manifest{
+				SonobuoyConfig: manifest.SonobuoyConfig{
+					ResultFormat: format,
+					ResultFile:   outputFile,
+				},
+			},
+		}}
+	}
+
+	testCases := []struct {
+		desc        string
+		plugin      plugin.Interface
+		expectedErr string
+
+		// key is used to lookup both the directory and the expected results.
+		key string
+	}{
+		{
+			desc:   "Job junit 1 file",
+			key:    "job-junit-1",
+			plugin: getPlugin("job", "junit", xmlFile),
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.desc, func(t *testing.T) {
+			item, err := ProcessRawPlugin(tc.plugin, tc.dir)
```

Comment:
Originally written when this method existed; I dont think it even does any more.